config/jobs-chromeos.yaml

```diff
@@ -75,6 +183,53 @@ _anchors:
         - kernel.Limits
         - kernel.PerfCallgraph
 
+  tast-mm-decode: &tast-mm-decode-job
+    <<: *tast-job
+    params:
+      tests:
+        - video.PlatformDecoding.ffmpeg_vaapi_vp9_0_group1_buf
+        - video.PlatformDecoding.ffmpeg_vaapi_vp9_0_group2_buf
+        - video.PlatformDecoding.ffmpeg_vaapi_vp9_0_group3_buf
+        - video.PlatformDecoding.ffmpeg_vaapi_vp9_0_group4_buf
+        - video.PlatformDecoding.ffmpeg_vaapi_vp9_0_level5_0_buf
+        - video.PlatformDecoding.ffmpeg_vaapi_vp9_0_level5_1_buf
+        - video.PlatformDecoding.ffmpeg_vaapi_av1
+        - video.PlatformDecoding.ffmpeg_vaapi_vp8_inter
+        - video.PlatformDecoding.ffmpeg_vaapi_h264_baseline
+        - video.PlatformDecoding.ffmpeg_vaapi_h264_main
+        - video.PlatformDecoding.ffmpeg_vaapi_hevc_main
+        - video.PlatformDecoding.vaapi_vp9_0_group1_buf
+        - video.PlatformDecoding.vaapi_vp9_0_group2_buf
+        - video.PlatformDecoding.vaapi_vp9_0_group3_buf
+        - video.PlatformDecoding.vaapi_vp9_0_group4_buf
+        - video.PlatformDecoding.vaapi_vp9_0_level5_0_buf
+        - video.PlatformDecoding.vaapi_vp9_0_level5_1_buf
+        - video.ChromeStackDecoder.*
+        - video.ChromeStackDecoderVerification.*
```

Comment:
I think we should create a new test suite to host video.ChromeStackDecoder.* and video.ChromeStackDecoderVerification.* - these were added in mm-decode in the legacy system but they would be better suited in a separate suite (we could call it `tast-decoder-chromestack`). These test should be run on trogdor (lazor, kingoftown), asurada and cherry for starters (we can extend them to the other mtk chromebooks at a later stage if they prove to be stable enough).

Reply:
`video.ChromeStackDecoder.*` is already in `tast-video` (and the only entry there) which is run on all chromebooks, so I'd suggest we add `video.ChromeStackDecoderVerification.*` there and drop both entries from `mm-decode`.

Reply:
Oh right, didn't see they were already defined there too. I still feel like `tast-video` is a bit generic though, I'd still suggest renaming this to `tast-decoder-chromestack` to hint at the fact that these are decoder tests and clearly distinguish them from the other video.* tast tests. Let me know if this makes sense to you!

Reply:
Makes sense yes, changed in new version